src/main/kotlin/app/ui/styles/Styles.kt

```diff
@@ -0,0 +1,42 @@
+package app.ui.styles
+
+import javafx.scene.Cursor
+import tornadofx.*
+
+class Styles: Stylesheet() {
```

Comment:
Styles of what?

Reply:
old file, not being used. I'll remove it